core/storage/feedback/gae_models.py

```diff
@@ -575,7 +564,7 @@ def get_most_recent_message(
         return message
 
     @classmethod
-    def get_message_count(cls, thread_id: str) -> int:
+    def get_message_count(cls, thread_id: str) -> Optional[int]:
```

Comment:
This seems off -- it shouldn't be optional (just from looking at the method name).